### DISCUSSION TITLE
Is postal itself dockerized?

### DISCUSSION THREAD

**Discussion by patchthecode** [postalserver/postal#2648] [Help with using Postal] [answered]
Im confused with the setup.

Here is what I was expecting, but correct me if im wrong.
I'm expecting to see:

Postal running inside its own container. 
I'm also expecting to see a docker-compose file that i can just run and have everything setup immediately (after some IP config modification). But it looks like Postal isnt running inside a container? I'm i correct?

**Comment by willpower232** [comment]
Postal is dockerised but relies on a few processes to accomplish everything. Have a look at the documentation and the installation repo.

https://github.com/postalserver/install
https://docs.postalserver.io/other/containers

**Comment by patchthecode** [comment]
The part that confuses me is the instructions here

<img width="706" alt="Screenshot 2023-10-19 at 4 34 16 PM" src="https://github.com/postalserver/postal/assets/2439146/b719a71f-9e58-4202-bd16-872100255048">


Which is why I asked the first question.
It looks like its not just a simple `docker compose up` command which sets everything up. It looks like there needs to be some setup on the host system to get it going.

The reason why I am asking this is because since there is now external set to be done, I cant just manage it easily like the other docker projects we're working on.  Is there a way it can be done fully using docker? If yes, let me know and we will try to find a solution.

**Reply by patchthecode** [reply]
It even runs postal by doing the follow (which doesnt look like it involves docker)

<img width="706" alt="Screenshot 2023-10-19 at 4 41 48 PM" src="https://github.com/postalserver/postal/assets/2439146/e234629d-2b4c-4de3-9fff-482618d92d22">

**Reply by patchthecode** [reply]
I saw a project here on github that seems to do what i am asking --> https://github.com/tiredofit/docker-postal
But since its from some random fellow on github, and have not been updated in a while, Im not sure if i should trust it or not.

**Reply by slobodangacesa** [reply]
Postal command is actually just docker-compose wrapper. Once 'installed', you'll get docker-compose.yml in /opt/postal/install, and you could use that to de-compose whole stack needed to run Postal server.

**Reply by willpower232** [reply, marked as answer]
exactly so, you can run Postal however you like, as long as you're running all the commands listed in the docker-compose.yml, you could make your own or run it with docker directly

**Reply by patchthecode** [reply]
thank you
